metrics/test/test_identical_codeblocks.py

```diff
@@ -21,8 +21,12 @@ def setUp(self):
         "CPD doesn't exist in the environment",
     )
     def test_identical_codeblocks_given_file_returns_1(self):
-        result = len(self._identicalcode.run())
-        self.assertEqual(result, 1)
+        result = self._identicalcode.run()
+        self.assertEqual(len(result), 1)
+        first_code_block = (result[0][1][0][1], result[0][1][0][2])
+        second_code_block = (result[0][1][1][1], result[0][1][1][2])
```

Comment:
this looks scary

Reply:
I think its a symptom of needing a class to structure the results data in a more readable way :)